lib/trustev/case.rb

```
@@ -0,0 +1,149 @@
+require 'nokogiri'
+require 'rest-client'
+
+module Trustev
+  class Case
+    def initialize(applicant_hash)
+      @applicant_hash = applicant_hash
+    end
+
+    def post
+      response = resource.post(payload.to_json)
+      @response_hash = ::Services::DataConverter.case_response_to_hash(response)
+    end
+
+    def response_hash
+      @response_hash || {}
+    end
+
+    def error_code
+      te_response['ErrorCode']
+    end
+
+    def error_text
+      te_response['ErrorText']
+    end
+
+    def error?
+      error_code != CaseResponseErrorCodes::NO_ERROR
+    end
+
+    def risk
+      te_response['TEvRisk'] unless error?
```

Rather than an implicit `nil` return, might it be better to raise an error if trying to attempt to access these?